tests/physics/step-18-rotation_matrix.cc

```diff
@@ -581,7 +581,7 @@ namespace Step18
     ++timestep_no;
     pcout << "Timestep " << timestep_no << " at time " << present_time
           << std::endl;
-    for (unsigned int cycle=0; cycle<2; ++cycle)
```

Comment:
Well, that's part of the sacrifice needed to reduce run time under debug mode! Its not really important in these examples anyway since an error in the rotation matrix would be picked up in coarse-level calculations anyway.